ui/src/components/setlist/SetlistFolderDrawer.tsx

```diff
@@ -42,6 +44,58 @@ interface SetlistFolderDrawerProps {
   mode: string;
 }
 
+interface SnackbarState {
+  open: boolean;
+  message: string;
+  severity: 'success' | 'error' | 'warning' | 'info';
+}
+
+interface ConfirmationDialogProps {
+  open: boolean;
+  onClose: () => void;
+  onConfirm: () => void;
+  title: string;
+  message: string;
+  confirmText?: string;
+  cancelText?: string;
+  confirmColor?: 'primary' | 'error' | 'warning' | 'success';
+}
+
+// Reusable Confirmation Dialog Component
+const ConfirmationDialog = ({
+  open,
+  onClose,
+  onConfirm,
+  title,
+  message,
+  confirmText = 'Confirm',
+  cancelText = 'Cancel',
+  confirmColor = 'primary',
+}: ConfirmationDialogProps) => (
+  <Dialog
+    open={open}
+    onClose={onClose}
+    aria-labelledby="confirmation-dialog-title"
+    aria-describedby="confirmation-dialog-description"
+    PaperProps={{
+      sx: { width: '30rem', borderRadius: '1.75rem', padding: '0.5rem' },
+    }}
+  >
+    <DialogTitle id="confirmation-dialog-title">{title}</DialogTitle>
+    <DialogContent>
+      <DialogContentText id="confirmation-dialog-description">{message}</DialogContentText>
+    </DialogContent>
+    <DialogActions>
+      <Button onClick={onClose} color="primary">
+        {cancelText}
+      </Button>
+      <Button onClick={onConfirm} color={confirmColor} autoFocus>
+        {confirmText}
+      </Button>
+    </DialogActions>
+  </Dialog>
```

Comment:
can you align the color of the dialog with what we have? I think you can copy from the drawer. (i.e. the purple, red, dialog bs, text color)

Reply:
let me move over the styling changes from the other PR